pallets/staking/src/inflation.rs

```diff
@@ -121,13 +120,16 @@ mod test {
         assert_payout(75_000, YEAR, 2_733);
         assert_payout(95_000, YEAR, 2_513);
         assert_payout(100_000, YEAR, 2_505);
+        const DAY: u64 = 24 * 60 * 60 * 1000;
```

Comment:
```suggestion
        const DAY: u64 = 24 * HOUR;
```

Reply:
The goal here is to minimize the diff to substrate [3.0.0](https://github.com/paritytech/substrate/blob/v3.0.0/frame/staking/src/lib.rs) so keeping as-is. (Although I agree with you...)